docs/src/app/components/Master.js

```diff
@@ -22,15 +23,18 @@ class Master extends Component {
 
   static childContextTypes = {
     muiTheme: PropTypes.object,
+    uniqueIdGen: PropTypes.func.isRequired,
   };
 
   state = {
     navDrawerOpen: false,
+    uniqueIdGen: getUniqueIdGenerator(),
```

Comment:
> Though docs site doesn't support server rendering currently.

Alright, let's keep it here. Supporting server-side rendering for a static documentation doesn't sounds like a bad idea 😁 .